Dockerfile

```diff
@@ -5,36 +5,45 @@ ARG BUILD_DATE
 ARG VERSION
 ARG THELOUNGE_VERSION
 LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DATE}"
-LABEL maintainer="aptalca"
+LABEL maintainer="aptalca,nemchik"
 
 # environment settings
 ENV THELOUNGE_HOME="/config" \
 NPM_CONFIG_LOGLEVEL="info"
 
 RUN \
- echo "**** install runtime packages ****" && \
- apk add --no-cache \
-	curl \
-	jq \
-	nodejs-npm && \
- npm config set unsafe-perm true && \
- echo "**** install the lounge irc ****" && \
- if [ -z ${THELOUNGE_VERSION+x} ]; then \
-	THELOUNGE_VERSION=$(curl -sL https://replicate.npmjs.com/registry/thelounge \
-	|jq -r '. | .["dist-tags"].latest'); \
- fi && \
- mkdir -p \
-	/app && \
- cd /app && \
- npm install -g \
-	thelounge@${THELOUNGE_VERSION} && \
- echo "**** ensure public true on startup aka no users ****" && \
- sed -i "s/public: false,/public: true,/g" /usr/lib/node_modules/thelounge/defaults/config.js && \
- echo "**** cleanup ****" && \
- rm -rf \
-	/root && \
- mkdir -p / \
-	/root
+	echo "**** install build packages ****" && \
```

Comment:
These are all tabs

Reply:
Resolved in https://github.com/linuxserver/docker-thelounge/pull/37/commits/b0388d0dcc15b932aa2e336e0acb90fdf3d4e547